sdk/kiosk/src/utils.ts

```diff
@@ -139,6 +138,23 @@ export function attachListingsAndPrices(
 	});
 }
 
+/**
+ * A helper that attaches the listing prices to kiosk listings.
+ */
+export function attachObjects(kioskData: KioskData, objects: SuiObjectData[]) {
```

Comment:
I really don't love that this helper mutates the inputs but it's not a dealbreaker.